logcall/__openerp__.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
```

Comment:
On 08/26/2016 01:32 AM, Eric Caudal wrote:
>
> In logcall/__openerp__.py
> <https://github.com/OCA/connector-telephony/pull/101#discussion_r76375612>:
>
> > @@ -0,0 +1,41 @@
> > +# -*- coding: utf-8 -*-
>
> I mean that we might want to improve the file name following Odoo
> recommendations:
> https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#modules
> Starting with the standard base odoo module hence my suggestion
> (crm_logcall). Nevertheless later I saw there is a crm_voip_logcall
> and now I am confused about the separation between both.
>
The logcall module was originally written by credative.co.uk. I believe
there is a crm_voip module in 8.0 if the module has been purchased and
part of 9.0 enterprise. This module is simply a way to bridge the two.

Trever